htmd/queues/acecloudqueue.py

```diff
@@ -91,7 +92,7 @@ def submit(self, dirs):
                 import hashlib
                 name = hashlib.sha256(os.path.abspath(d).encode('utf-8')).hexdigest()[:10]
 
-            runscript = self._getRunScript(d)
+            runscript = self._hackRunScript(d)
```

Comment:
:laughing:

Reply:
I know it's ugly, but it's easier than to "fix" the AceCloud image